helpers/skype_helper.py

```diff
@@ -0,0 +1,35 @@
+"""
+Helper module for Skype
+"""
+import os
+import sys
+import requests
+
+# add project root to sys path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from helpers.base_helper import BaseHelper
+from conf import skype_conf
+
+class SkypeHelper(BaseHelper):
+    """
+    Skype Helper object
+    """
+
+    def post_message_on_skype(self, message):
```

Comment:
Method signature should be smart enough to take whatever might be configurable as an argument.